src/net/sourceforge/kolmafia/swingui/panel/GenericPanel.java

```diff
@@ -35,7 +35,7 @@
   protected ConfirmedListener CONFIRM_LISTENER = new ConfirmedListener();
   protected CancelledListener CANCEL_LISTENER = new CancelledListener();
 
-  protected HashMap listenerMap;
+  protected HashMap<Object, WeakReference<ActionConfirmListener>> listenerMap;
```

Comment:
Should this be JTextComponent instead of Object?

Reply:
I didn't notice that indeed! Nice catch